src/Run/Setup.cs

```diff
@@ -57,6 +93,35 @@ private string ParseSettingValue(string inputValue)
             return value;
         }
 
+        private void ParseRunToolSettings(string commandSelectedByUser = null)
+        {
+            Setting tempSetting;
+            Command tempCommand;
+            foreach(var toolSetting in ToolSettings.Keys.Select(k => k.ToString()).ToArray())
+            {
+                string tempValue = null;
+                // Attempt to get run tool setting value from parameters
+                SettingParameters.TryGetValue(toolSetting, out tempValue);
+                ToolSettings[toolSetting] = tempValue;
+
+                // Attempt to get run tool setting value from command section
+                if (string.IsNullOrEmpty(ToolSettings[toolSetting]) && commandSelectedByUser != null)
+                {
+                    if (Commands.TryGetValue(commandSelectedByUser, out tempCommand))
+                    {
+                        tempCommand.DefaultValues.Settings.TryGetValue(toolSetting, out tempValue);
+                        ToolSettings[toolSetting] = tempValue;
+                    }
+                }
+                // Attempt to get run tool setting value from settings section
+                if (string.IsNullOrEmpty(ToolSettings[toolSetting]))
+                {
+                    Settings.TryGetValue(toolSetting, out tempSetting);
```

Comment:
I completely missed this line: https://github.com/dotnet/buildtools/pull/980/files#diff-6f4e4ec63e7b18f5f61e1d0ba5c10087R104

Sorry :(